app/companies/create-job/page.tsx

```diff
@@ -52,13 +57,13 @@ export default function CreateJob() {
 
   //TODO: Put the following code in a Autosuggest Input component
   const languages = [
```

Comment:
Move this to global vars.

`NEXT_PUBLIC_SKILLS=Javascript,Solidity,SQL`

Rename to `skills`.